kiwi/repository/yum.py

```diff
@@ -90,6 +90,16 @@ def post_init(self, custom_args=None):
         self._create_runtime_plugin_config_parser()
         self._write_runtime_config()
 
+    def setup_package_database_configuration(self):
+        """
+        Setup package database configuration
+
+        For the supported RHEL versions there is no rpm database
+        setup required. We support building RHEL images on RHEL
+        host systems only at the moment.
+        """
```

Comment:
We do? Why?

Reply:
because I thought dnf will be the successor, not correct ?

Reply:
ok, I've added the macro setup there too